### PR TITLE
Added a step to the ES6 recipe to make sure we don't have JS files du…

### LINKED ISSUE #392: Adding ES2015 support duplicates the content in app.js
Downloading PSK 1.1.0 and following the steps to add support for ES2015 results in the content of app.js  being duplicated.
No code changes, nothing other than adding the changes to gulpfile.js.

### PR DESCRIPTION
**Pull request by nicolasgarnier**
…plicated.

Also made steps clearer by adding some inline comments and more context.

Fixes #392

### CODE REVIEW

File: docs/add-es2015-support-babel.md [hunk 1]
```diff
@@ -44,10 +44,10 @@ gulp.task('serve', ['styles', 'elements', 'images', 'js'], function () {
 
   ...
 
-  gulp.watch(['app/**/*.html'], ['js', reload]);
+  gulp.watch(['app/**/*.html'], ['js', reload]); // Added 'js' and the array here!
```
Comment (samccone) [inline docs/add-es2015-support-babel.md]: we could use the 

```
```patch
```

syntax here?

Comment (nicolasgarnier) [inline docs/add-es2015-support-babel.md]: Woo. I've actually been looking for this for a while :D Let me try that...

**Comment by nicolasgarnier** [comment]
Switched to DIFF syntax. PTAL

**Comment by samccone** [comment]
:boom: SO MUCH BETTER

**Pull request merged**